src/server/database/knexfile.js

```diff
@@ -9,11 +10,22 @@ export default {
 
   connection: {
     host: process.env.POSTGRES_HOST,
-    database: process.env.POSTGRES_DATABASE,
+    database:
+      process.env.NODE_ENV === 'test'
+        ? 'zola-test'
+        : process.env.POSTGRES_DATABASE,
     user: process.env.POSTGRES_USER,
     password: process.env.POSTGRES_PASSWORD
   },
 
+  migrations: {
+    directory: path.join(__dirname, 'migrations')
+  },
+
+  seeds: {
+    directory: path.join(__dirname, `seeds/${process.env.NODE_ENV}`)
```

Comment:
That way we can have different seeds for each environment simply by adding files in a folder. :)